src/frontend/api/fagsak.ts

```diff
@@ -94,14 +95,32 @@ export const apiOpprettEllerOppdaterVedtak = (fagsakId: number, data: IRestVilk
 };
 
 export interface IOpprettBeregningData {
-    barnasBeregning: IBarnBeregning[];
+    personBeregninger: IPersonBeregning[];
 }
 
-export const apiOpprettBeregning = (fagsak: IFagsak, data: any) => {
+export const apiOpprettBeregning = (fagsak: IFagsak, data: IOpprettBeregningData) => {
+    const dataTilKalkulator = data.personBeregninger.map(beregning => ({
+        personident: beregning.personident,
+        ytelsetype: beregning.ytelseType,
+        halvytelse: beregning.deltYtelse,
+        stønadFom: formaterIsoDato(beregning.stønadFom, datoformat.ISO_MÅNED),
+        stønadTom: formaterIsoDato(beregning.stønadTom, datoformat.ISO_MÅNED),
```

Comment:
Hvorfor trenger vi en tom? Kan den ikke utledes backend.

Reply:
I en del caser kan den nok utledes backend, men vet ikke om den alltid kan det. Tok den med her nå kun for å sikre at vi får all dataen vi trenger. @nav-jorn-ola-birkeland Har du noen innspill her? Kan protokollen til kalkulatoren klare seg uten en tom?